test/integration/cluster/test_cluster_show.py

```diff
@@ -3,7 +3,9 @@
 import subprocess
 
 import pytest
-from etcd_helper import etcd_password, etcd_username
+from integration.connect.test_connect import is_tarantool_ee
```

Comment:
Please, move the is_tarantool_ee to `utils` test submodule.

Reply:
Moved it to `utils`.